src/Questionnaire/Questionnaire.js

```diff
@@ -70,3 +97,15 @@ const ButtonRow = styled.section`
   display: flex;
   justify-content: space-between;
 `
+
+const H2Styled = styled.h2`
+  margin-top: -10px;
+`
+const QuestionnaireButton = styled(Button)`
+  ${(props) =>
+    props.borderColor ? 'border: 2px solid ' + props.borderColor + ';' : ''}
```

Comment:
Could this work?
```suggestion
  ${(props) =>
    props.borderColor && 'border: 2px solid ' + props.borderColor + ';'}
```
do you need the " '' "

Reply:
No, you are right, I use too many ternaries :). I changed it to you suggestion!